runtime/jcl/common/jclcinit.c

```diff
@@ -68,86 +68,61 @@ jint computeFullVersionString(J9JavaVM* vm)
 	PORT_ACCESS_FROM_JAVAVM(vm);
 	const char *osarch;
 	const char *osname;
-	const char *vmVersion = NULL;
-	const char *gcVersion = NULL;
+	const char *j2se_version_info;
```

Comment:
All local variables should be initialized; please also initialize `osarch` and `osname'.